src/test/unit_tests/test_s3_bucket_policy.js

```diff
@@ -221,7 +221,8 @@ mocha.describe('s3_bucket_policy', function() {
             Bucket: BKT,
             Policy: JSON.stringify(policy)
         });
-        const res_a = await s3_a.getBucketPolicy({ // should work - user a has get_bucket_policy permission
+        // Bucket Policy Should Be Managed Only by Bucket Owner : #8477
```

Comment:
I'm not sure we want to leave the issue link in our code (#8477).

Reply:
added this comment because of the removed comment, I can remove the issue id